File: drs/src/test/com/db4o/drs/test/ByteArrayHolder.java
package com.db4o.drs.test;

public class ByteArrayHolder implements IByteArrayHolder {
	
	private byte[] _bytes;
	
	public ByteArrayHolder() {
	}
	
	public ByteArrayHolder(byte[] bytes) {
		this._bytes = bytes;
	}
	
	public byte[] getBytes() {
		return _bytes;
	}

	public void setBytes(byte[] bytes) {
		_bytes = bytes;
	}
}

File: db4o.net/drs/Db4objects.Drs.Tests/Db4objects.Drs.Tests/ByteArrayHolder.cs
/* Copyright (C) 2004 - 2008  db4objects Inc.  http://www.db4o.com */

using Db4objects.Drs.Tests;

namespace Db4objects.Drs.Tests
{
	public class ByteArrayHolder : IIByteArrayHolder
	{
		private byte[] _bytes;

		public ByteArrayHolder()
		{
		}

		public ByteArrayHolder(byte[] bytes)
		{
			this._bytes = bytes;
		}

		public virtual byte[] GetBytes()
		{
			return _bytes;
		}

		public virtual void SetBytes(byte[] bytes)
		{
			_bytes = bytes;
		}
	}
}
